package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ocapi-client",
-  "version": "0.2.2",
+  "name": "commercecloud-ocapi-client",
+  "version": "0.0.1",
   "description": "An ES6 JavaScript Client for Salesforce Open Commerce API",
```

Comment:
So is the `lib/index.js` transpiled to ES5 or do we ship the library as ES6?

Reply:
Done and done